sdk/quantum/quantum-jobs/README.md

```diff
@@ -1,41 +1,209 @@
+
+  [Source code][source] | [API reference documentation](https://docs.microsoft.com/qsharp/api/) | [Product documentation](https://docs.microsoft.com/azure/quantum/)
+
 ## Getting started
 
+This section includes everything a developer needs to install and create their first client connection _very quickly_.
+
 ### Install the package
 
+Install the Azure Quantum Jobs client library for Javascript with `npm`:
+
 ```bash
 npm install @azure/quantum-jobs
 ```
 
 ### Prerequisites
 
 - Node.js version 8.x.x or higher
+- [Azure subscription](https://azure.microsoft.com/free/)
+- [Azure Quantum Workspace][workspaces]
+
+### Authenticate the client
+
+To authenticate with the service, you can use [DefaultAzureCredential](https://docs.microsoft.com/dotnet/api/azure.identity.defaultazurecredential?view=azure-dotnet). This will try different authentication mechanisms based on the environment (e.g. Environment Variables, ManagedIdentity, CachedTokens) and finally it will fallback to InteractiveBrowserCredential.
+
+The client also allows the user to override the above behavior by passing their own [TokenCredential](https://docs.microsoft.com/dotnet/api/azure.core.tokencredential?view=azure-dotnet).
+
+`TokenCredential` is the default Authentication mechanism used by Azure SDKs.
 
 ## Key concepts
 
-**TO BE FILLED**
+`QuantumJobClient` is the root class to be used to authenticate, and create, enumerate, and cancel jobs.
+
+`JobDetails` contains all the properties of a job.
+
+`ProviderStatus` contains status information for a provider.
+
+`QuantumJobQuota` contains quota properties.
 
 ## Examples
 
-**TO BE FILLED**
+### Create the client
+
+Create an instance of the QuantumJobClient by passing in these parameters:
+
+- [Subscription Id][subscriptions] - looks like XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX and can be found in your list of subscriptions on azure
+- [Resource Group Name][resource-groups] - a container that holds related resources for an Azure solution
+- [Workspace Name][workspaces] - a collection of assets associated with running quantum or optimization applications
+- [Location][location] - choose the best data center by geographical region
+- [Storage Container Name][blob-storage] - your blob storage
+- [Credential][credentials] - used to authenticate
+
+```Javascript Snippet
+    const credential = new DefaultAzureCredential();
+
+    // Create a QuantumJobClient
+    const subscriptionId = "your_subscription_id";
+    const resourceGroupName = "your_resource_group_name";
+    const workspaceName = "your_quantum_workspace_name";
+    const storageContainerName = "mycontainer";
+    const location = "westus"; //"your_location";
+    const endpoint = "https://" + location + ".quantum.azure.com";
+
+    const quantumJobClient = new QuantumJobClient(
+      credential,
+      subscriptionId,
+      resourceGroupName,
+      workspaceName,
+      {
+        endpoint: endpoint,
+        credentialScopes: "https://quantum.microsoft.com/.default"
```

Comment:
If this scope is a constant, why not make it internal to the SDK. Why ask users to pass it?

Reply:
I believe it will be defined in the SDK as part of the custom layer (in the next round)

Reply:
Ideally, the scope should have already been added in the generated code by autorest since we are passing 
```
add-credentials: true
credential-scopes: ["https://quantum.microsoft.com/.default"]
```

So it may be an autorest bug that I'll follow-up next week, or as Sarangan said,  add it in the custom layer.